.github/workflows/scripts/spdx-tools-java-wrapper.sh

```diff
@@ -40,18 +41,35 @@ verify() {
         echo "Writing local job summary to '$GITHUB_STEP_SUMMARY'."
     fi
 
-    # TODO: Limit this to only the files modified in a PR.
-    find analysed-packages/ -iname '*.spdx' -print0 | \
-        xargs -0 -P8 -I {} \
-        bash -c "'$javabin' -jar '$jar' Verify {} || echo '* [{}]($GITHUB_SERVER_URL/$GITHUB_REPOSITORY/blob/$GITHUB_SHA/{})' >> '$GITHUB_STEP_SUMMARY'"
-
-    if [[ -s "$GITHUB_STEP_SUMMARY" ]]; then
-        local count=$(wc -l < "$GITHUB_STEP_SUMMARY")
-        echo -e "### The following $count \`.spdx\` files are invalid :x: (see the job logs for details)\n$(cat $GITHUB_STEP_SUMMARY)" > "$GITHUB_STEP_SUMMARY"
-        exit 1
-    else
-        echo "### All \`.spdx\` files are valid :heavy_check_mark:" > "$GITHUB_STEP_SUMMARY"
+    for package in "$@"; do
+        echo "Analysing package $package"
+        if $javabin -jar "$jar" Verify $package; then
+            # Package is valid. Generate conversion
+            convert "$package"
+        else
+            echo -e "### The following $package are invalid"
+            exit 1
```

Comment:
this fails on first invalid file?

Reply:
previously it was "collect all failures, report them, and fail then"

Reply:
Yes, i know. but is just standard CI workflow. If you think we should report any fails, we can restore this behaviour.
But my logic is:

If you build a C++ project, and one of the files fail to build, everything stops. It not continues compiling untils catch everything.

I honestly think in fail soon, catch soon.